app/apollo/resolvers/group.js

```diff
@@ -399,6 +411,9 @@ const groupResolvers = {
 
         pubSub.channelSubChangedFunc({org_id}, context);
 
+        // Allow graphQL plugins to retrieve more information
+        context.pluginContext = {clusterNames: clusterIds, groupUuids: groupUuids};
```

Comment:
`clusterNames: clusterIds`: names or uuids?  we want/need both for the context to be usable, right?